docs/guides/developer/docs/api/series-api.md

```diff
@@ -461,11 +461,12 @@ __Example__
 
 Updates a series' access policy.
 
-Parameters | Required | Type                  | Description
-:----------|:---------|:----------------------|:-----------
-`acl`      | yes      | [`acl`](types.md#acl) | Access policy to be applied
+Parameters | Required | Type                        | Description                 | Default | Version
+:----------|:---------|:----------------------------|:----------------------------|:--------|:-------
+`acl`      | yes      | [`acl`](types.md#acl)       | Access policy to be applied |         |
+`override` | no       | [`boolean`](types.md#basic) | Whether the episode ACL of all events of this series should be removed | false | 1.2.0
```

Comment:
@KatrinIhler I would rather stay with appending " (version 1.2.0 or higher)" to the description (as other examples in the same page) as most fields are available for all External API versions. If we would consistently add a column version to all fields, most values would be 1.1.0 or higher.

What do you think?

Reply:
Actually this is already inconsistent in the external api documentation (there is both this and what you described) and I think this is a bit more clean, but I don't mind either way...